<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>JACK Audio Connection Kit - Qt GUI Interface.

   Copyright (C) 2003-2025, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qjackctlGraphForm</class>
 <widget class="QMainWindow" name="qjackctlGraphForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Graph</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qjackctl.qrc" >:/images/graph1.png</iconset>
  </property>
  <widget class="QWidget" name="MainCentralWidget">
   <layout class="QVBoxLayout">
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item>
     <widget class="qjackctlGraphCanvas" name="graphCanvas"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="graphMenu">
    <property name="title">
     <string>&amp;Graph</string>
    </property>
    <addaction name="graphConnectAction"/>
    <addaction name="graphDisconnectAction"/>
    <addaction name="separator"/>
    <addaction name="graphCloseAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="editSelectAllAction"/>
    <addaction name="editSelectNoneAction"/>
    <addaction name="editSelectInvertAction"/>
    <addaction name="separator"/>
    <addaction name="editSearchItemAction"/>
    <addaction name="editRenameItemAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="viewThumbviewMenu">
     <property name="title">
      <string>T&amp;humbview</string>
     </property>
     <property name="icon">
      <iconset resource="qjackctl.qrc">:/images/graphThumbview.png</iconset>
     </property>
     <addaction name="viewThumbviewTopLeftAction"/>
     <addaction name="viewThumbviewTopRightAction"/>
     <addaction name="viewThumbviewBottomLeftAction"/>
     <addaction name="viewThumbviewBottomRightAction"/>
     <addaction name="separator"/>
     <addaction name="viewThumbviewNoneAction"/>
    </widget>
    <widget class="QMenu" name="viewZoomMenu">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <property name="icon">
      <iconset resource="qjackctl.qrc">:/images/graphZoomTool.png</iconset>
     </property>
     <addaction name="viewZoomInAction"/>
     <addaction name="viewZoomOutAction"/>
     <addaction name="separator"/>
     <addaction name="viewZoomFitAction"/>
     <addaction name="viewZoomResetAction"/>
    </widget>
    <widget class="QMenu" name="viewColorsMenu">
     <property name="title">
      <string>Co&amp;lors</string>
     </property>
     <property name="icon">
      <iconset resource="qjackctl.qrc">:/images/graphColors.png</iconset>
     </property>
     <addaction name="viewColorsJackAudioAction"/>
     <addaction name="viewColorsJackMidiAction"/>
     <addaction name="separator"/>
     <addaction name="viewColorsResetAction"/>
    </widget>
    <widget class="QMenu" name="viewSortMenu">
     <property name="title">
      <string>S&amp;ort</string>
     </property>
     <addaction name="viewSortPortNameAction"/>
     <addaction name="viewSortPortTitleAction"/>
     <addaction name="viewSortPortIndexAction"/>
     <addaction name="separator"/>
     <addaction name="viewSortAscendingAction"/>
     <addaction name="viewSortDescendingAction"/>
    </widget>
    <addaction name="viewMenubarAction"/>
    <addaction name="viewToolbarAction"/>
    <addaction name="viewStatusbarAction"/>
    <addaction name="separator"/>
    <addaction name="viewThumbviewMenu"/>
    <addaction name="separator"/>
    <addaction name="viewTextBesideIconsAction"/>
    <addaction name="viewColorsMenu"/>
    <addaction name="viewSortMenu"/>
    <addaction name="separator"/>
    <addaction name="viewCenterAction"/>
    <addaction name="viewZoomMenu"/>
    <addaction name="viewZoomRangeAction"/>
    <addaction name="separator"/>
    <addaction name="viewRepelOverlappingNodesAction"/>
    <addaction name="viewConnectThroughNodesAction"/>
    <addaction name="separator"/>
    <addaction name="viewRefreshAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction"/>
    <addaction name="helpAboutQtAction"/>
   </widget>
   <addaction name="graphMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="separator"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="graphConnectAction"/>
   <addaction name="graphDisconnectAction"/>
   <addaction name="separator"/>
   <addaction name="viewCenterAction"/>
   <addaction name="separator"/>
   <addaction name="viewZoomInAction"/>
   <addaction name="viewZoomOutAction"/>
   <addaction name="separator"/>
   <addaction name="viewZoomFitAction"/>
   <addaction name="viewZoomResetAction"/>
   <addaction name="separator"/>
   <addaction name="viewZoomRangeAction"/>
  </widget>
  <action name="graphConnectAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphConnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="iconText">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
   <property name="statusTip">
    <string>Connect selected ports</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="graphDisconnectAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphDisconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="iconText">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
   <property name="statusTip">
    <string>Disconnect selected ports</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="graphCloseAction">
   <property name="text">
    <string>Cl&amp;ose</string>
   </property>
   <property name="iconText">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close this application window</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="editSelectAllAction">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="iconText">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
   <property name="statusTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="editSelectNoneAction">
   <property name="text">
    <string>Select &amp;None</string>
   </property>
   <property name="iconText">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Select None</string>
   </property>
   <property name="statusTip">
    <string>Select None</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="editSelectInvertAction">
   <property name="text">
    <string>Select &amp;Invert</string>
   </property>
   <property name="iconText">
    <string>Select Invert</string>
   </property>
   <property name="toolTip">
    <string>Select Invert</string>
   </property>
   <property name="statusTip">
    <string>Select Invert</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="editRenameItemAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphRename.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rename...</string>
   </property>
   <property name="iconText">
    <string>Rename item</string>
   </property>
   <property name="toolTip">
    <string>Rename Item</string>
   </property>
   <property name="statusTip">
    <string>Rename Item</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="editSearchItemAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphFind.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="iconText">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find nodes</string>
   </property>
   <property name="statusTip">
    <string>Find nodes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="viewMenubarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Menubar</string>
   </property>
   <property name="iconText">
    <string>Menubar</string>
   </property>
   <property name="toolTip">
    <string>Menubar</string>
   </property>
   <property name="statusTip">
    <string>Show/hide the main program window menubar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="viewToolbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Toolbar</string>
   </property>
   <property name="iconText">
    <string>Toolbar</string>
   </property>
   <property name="toolTip">
    <string>Toolbar</string>
   </property>
   <property name="statusTip">
    <string>Show/hide main program window file toolbar</string>
   </property>
  </action>
  <action name="viewStatusbarAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Statusbar</string>
   </property>
   <property name="iconText">
    <string>Statusbar</string>
   </property>
   <property name="toolTip">
    <string>Statusbar</string>
   </property>
   <property name="statusTip">
    <string>Show/hide the main program window statusbar</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewThumbviewTopLeftAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Top Left</string>
   </property>
   <property name="iconText">
    <string>Top left</string>
   </property>
   <property name="toolTip">
    <string>Top left</string>
   </property>
   <property name="statusTip">
    <string>Show the thumbnail overview on the top-left</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewThumbviewTopRightAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top &amp;Right</string>
   </property>
   <property name="iconText">
    <string>Top right</string>
   </property>
   <property name="toolTip">
    <string>Top right</string>
   </property>
   <property name="statusTip">
    <string>Show the thumbnail overview on the top-right</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewThumbviewBottomLeftAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bottom &amp;Left</string>
   </property>
   <property name="iconText">
    <string>Bottom Left</string>
   </property>
   <property name="toolTip">
    <string>Bottom left</string>
   </property>
   <property name="statusTip">
    <string>Show the thumbnail overview on the bottom-left</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewThumbviewBottomRightAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Bottom Right</string>
   </property>
   <property name="iconText">
    <string>Bottom right</string>
   </property>
   <property name="toolTip">
    <string>Bottom right</string>
   </property>
   <property name="statusTip">
    <string>Show the thumbnail overview on the bottom-right</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewThumbviewNoneAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;None</string>
   </property>
   <property name="iconText">
    <string>None</string>
   </property>
   <property name="toolTip">
    <string>Hide thumbview</string>
   </property>
   <property name="statusTip">
    <string>Hide the thumbnail overview</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewTextBesideIconsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Text Beside &amp;Icons</string>
   </property>
   <property name="iconText">
    <string>Text beside icons</string>
   </property>
   <property name="toolTip">
    <string>Text beside icons</string>
   </property>
   <property name="statusTip">
    <string>Show/hide text beside icons</string>
   </property>
  </action>
  <action name="viewCenterAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphCenter.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Center</string>
   </property>
   <property name="iconText">
    <string>Center</string>
   </property>
   <property name="toolTip">
    <string>Center</string>
   </property>
   <property name="statusTip">
    <string>Center view</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewRefreshAction">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="iconText">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="statusTip">
    <string>Refresh view</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="viewZoomInAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="iconText">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="statusTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="viewZoomOutAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphZoomOut.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="iconText">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="statusTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="viewZoomFitAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphZoomFit.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Fit</string>
   </property>
   <property name="iconText">
    <string>Zoom Fit</string>
   </property>
   <property name="toolTip">
    <string>Zoom Fit</string>
   </property>
   <property name="statusTip">
    <string>Zoom Fit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="viewZoomResetAction">
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphZoomReset.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Reset</string>
   </property>
   <property name="iconText">
    <string>Zoom Reset</string>
   </property>
   <property name="toolTip">
    <string>Zoom Reset</string>
   </property>
   <property name="statusTip">
    <string>Zoom Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="viewZoomRangeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qjackctl.qrc">:/images/graphZoomRange.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom Range</string>
   </property>
   <property name="iconText">
    <string>Zoom Range</string>
   </property>
   <property name="toolTip">
    <string>Zoom Range</string>
   </property>
   <property name="statusTip">
    <string>Zoom Range</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewColorsJackAudioAction">
   <property name="text">
    <string>JACK &amp;Audio...</string>
   </property>
   <property name="iconText">
    <string>JACK Audio color</string>
   </property>
   <property name="toolTip">
    <string>JACK Audio color</string>
   </property>
   <property name="statusTip">
    <string>JACK Audio color</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewColorsJackMidiAction">
   <property name="text">
    <string>JACK &amp;MIDI...</string>
   </property>
   <property name="iconText">
    <string>JACK MIDI</string>
   </property>
   <property name="toolTip">
    <string>JACK MIDI color</string>
   </property>
   <property name="statusTip">
    <string>JACK MIDI color</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewColorsAlsaMidiAction">
   <property name="text">
    <string>ALSA M&amp;IDI...</string>
   </property>
   <property name="iconText">
    <string>ALSA MIDI</string>
   </property>
   <property name="toolTip">
    <string>ALSA MIDI color</string>
   </property>
   <property name="statusTip">
    <string>ALSA MIDI color</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewColorsJackCvAction">
   <property name="text">
    <string>JACK &amp;CV...</string>
   </property>
   <property name="iconText">
    <string>JACK CV color</string>
   </property>
   <property name="toolTip">
    <string>JACK CV color</string>
   </property>
   <property name="statusTip">
    <string>JACK CV color</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewColorsJackOscAction">
   <property name="text">
    <string>JACK &amp;OSC...</string>
   </property>
   <property name="iconText">
    <string>JACK OSC</string>
   </property>
   <property name="toolTip">
    <string>JACK OSC color</string>
   </property>
   <property name="statusTip">
    <string>JACK OSC color</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewColorsResetAction">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="iconText">
    <string>Reset colors</string>
   </property>
   <property name="toolTip">
    <string>Reset colors</string>
   </property>
   <property name="statusTip">
    <string>Reset colors</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewSortPortNameAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Port &amp;Name</string>
   </property>
   <property name="toolTip">
    <string>Port name</string>
   </property>
   <property name="statusTip">
    <string>Sort by port name</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewSortPortTitleAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Port &amp;Title</string>
   </property>
   <property name="toolTip">
    <string>Port title</string>
   </property>
   <property name="statusTip">
    <string>Sort by port title</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewSortPortIndexAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Port &amp;Index</string>
   </property>
   <property name="toolTip">
    <string>Port index</string>
   </property>
   <property name="statusTip">
    <string>Sort by port index</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewSortAscendingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Ascending</string>
   </property>
   <property name="toolTip">
    <string>Ascending</string>
   </property>
   <property name="statusTip">
    <string>Ascending sort order</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewSortDescendingAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Descending</string>
   </property>
   <property name="toolTip">
    <string>Descending</string>
   </property>
   <property name="statusTip">
    <string>Descending sort order</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewRepelOverlappingNodesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repel O&amp;verlapping Nodes</string>
   </property>
   <property name="iconText">
    <string>Repel nodes</string>
   </property>
   <property name="toolTip">
    <string>Repel overlapping nodes</string>
   </property>
   <property name="statusTip">
    <string>Repel overlapping nodes</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="viewConnectThroughNodesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect Thro&amp;ugh Nodes</string>
   </property>
   <property name="iconText">
    <string>Connect Through Nodes</string>
   </property>
   <property name="toolTip">
    <string>Connect through nodes</string>
   </property>
   <property name="statusTip">
    <string>Whether to draw connectors through or around nodes</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="helpAboutAction">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="iconText">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Show information about this application program</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="helpAboutQtAction">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
   <property name="iconText">
    <string>About Qt...</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>Show information about the Qt toolkit</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>qjackctlGraphCanvas</class>
   <extends>QGraphicsView</extends>
   <header>qjackctlGraph.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qjackctl.qrc"/>
 </resources>
 <connections/>
</ui>
